.github/workflows/loadtest-europe-client-deploy.yml

```diff
@@ -0,0 +1,91 @@
+name: "[LOADTEST-CLIENT-EUROPE] Deploy to Europe Loadtest Client"
+on:
+  push:
+      branches:
+        - 1115-add-new-server-deployment-workflow
+  workflow_dispatch:
+    inputs:
+      target_server:
+        type: choice
+        description: Loadtest target server
+        options:
+          - Europe
+          - Chile
+
+jobs:
+  build-deploy:
+    name: Build and deploy to Europe Loadtest Client
+    runs-on: ubuntu-22.04
+    environment:
+      name: testing-europe
+      url: https://loadtest-europe-client.championsofmirra.com/
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Tailscale
+        uses: tailscale/github-action@v2
+        with:
+          oauth-client-id: ${{ secrets.TS_OAUTH_CLIENT_ID }}
+          oauth-secret: ${{ secrets.TS_OAUTH_SECRET }}
+          tags: tag:ci
+
+      - name: Create ssh private key file from env var
+        env:
+          SSH_KEY: ${{ secrets.SSH_KEY }}
+          SSH_HOST: ${{ vars.TS_LOADTEST_CLIENT_HOST }}
+        run: |
+          set -ex
+          mkdir -p ~/.ssh/
+          sed -E 's/(-+(BEGIN|END) OPENSSH PRIVATE KEY-+) *| +/\1\n/g' <<< "$SSH_KEY" > ~/.ssh/id_ed25519
+          chmod 400 ~/.ssh/id_ed25519
+          retries=5; until ssh-keyscan $SSH_HOST >> ~/.ssh/known_hosts || [ $retries -eq 0 ]; do ((retries--)); sleep 5; done
+
+      - name: Copy deploy script
+        env:
+          SSH_USERNAME: ${{ vars.SSH_LOADTEST_USERNAME }}
+          SSH_HOST: ${{ vars.TS_LOADTEST_CLIENT_HOST }}
+        run: |
+          set -ex
+          rsync -avz --mkpath devops/deploy.sh ${SSH_USERNAME}@${SSH_HOST}:/home/${SSH_USERNAME}/deploy-script/
+
+      - name: Execute deploy script
+        env:
+          XDG_RUNTIME_DIR: /run/user/1001 # Temporary fix for systemd commands.
+          SSH_HOST: ${{ vars.TS_LOADTEST_CLIENT_HOST }}
+          SSH_USERNAME: ${{ vars.SSH_LOADTEST_USERNAME }}
+          MIX_ENV: ${{ vars.MIX_ENV }}
+          RELEASE: arena_load_test
+          TARGET_SERVER: ${{ inputs.target_server }}
+          LOADTEST_EUROPE_HOST: ${{ vars.LOADTEST_EUROPE_HOST }}
+          LOADTEST_CHILE_HOST: ${{ vars.LOADTEST_CHILE_HOST }}
+          PHX_SERVER: ${{ vars.PHX_SERVER }}
+          PHX_HOST: ${{ vars.LOADTEST_CLIENT_HOST }}
+          PORT: ${{ vars.ARENA_PORT }}
+          BOT_MANAGER_PORT: ${{ vars.BOT_MANAGER_PORT }}
+          DATABASE_URL: ${{ secrets.DATABASE_URL }}
+          SECRET_KEY_BASE: ${{ secrets.SECRET_KEY_BASE }}
+          NEWRELIC_APP_NAME: ${{ vars.NEWRELIC_APP_NAME_LOADTEST_CLIENT }}
+          NEWRELIC_KEY: ${{ secrets.NEWRELIC_KEY }}
+          DBUS_SESSION_BUS_ADDRESS: ${{ vars.DBUS_SESSION_BUS_ADDRESS }}
+          BRANCH_NAME: 1115-add-new-server-deployment-workflow
```

Comment:
would we ever need the configurator to be deployed in those servers? If so we'd need to export `CONFIGURATOR_DATABASE_URL` as well, no?